scripts/cangen/main.py

```diff
@@ -33,6 +33,9 @@ def parse():
     parser.add_argument(
         "--project", type=str, required=True, help="Name of the project"
     )
+    parser.add_argument(
+        "--log-level", "-l", dest='level', choices=["STATUS", "INFO", "VERBOSE", "DEBUG"], default="INFO", help="Log verbosity threshold"
```

Comment:
Since this script is called automatically, I'd rather not have a shorthand `-l` option. Removing this forces the build system developer to write the full `--log-level` in the generator script which is clearer for someone reading it later. 

You can just remove the `"-l",` argument, argparse can have options without short forms.